src/parser.rs

```diff
@@ -2358,8 +2358,7 @@ impl<'a> Parser<'a> {
                         ]) // This couldn't possibly be a bad idea
                     })?
                     .into_iter()
-                    .filter(|i| i.is_some())
-                    .map(|i| i.unwrap())
+                    .flatten()
```

Comment:
Is it correct that  this is a real bug that was found/fixed with the fuzzer?

Reply:
no, just make clippy happy